### PR TITLE
Migrate token schema

### PR DESCRIPTION
**Pull request by h0ngcha0**
### Issue / feature description

Migrate the token schema

### Changes

Migrate the token schema to include verified

### Checklist

- [x] Rebased to the last commit of the target branch (or merged)
- [x] Code self-reviewed
- [x] Code self-tested
- [ ] Tests updated (if needed)
- [ ] All tests are passing locally

### Manual Tests

- [x] Transfer ALPH, Token, and NFT
- [x] Display of Token/NFT metadata
- [x] Works with the simple dApp within the repository (yarn run dev)
- [x] Ledger support

Please keep your pull request as small as possible. If you need to make multiple changes, please create separate pull requests for each change. Create a draft pull request if you need early feedback. This will mark the pull request as a work in progress and prevent it from being reviewed or merged until you are ready.

Please move drafts to the ready for review (regular PR) when you are ready to start the review process and other developers will pick it up from there.

### CODE REVIEW

File: packages/extension/src/shared/token/storage.ts
Comment (Lbqds) [inline packages/extension/src/shared/token/storage.ts:55]: Do we need to add the `verified` to the schema? https://github.com/alephium/extension-wallet/blob/3724c939558f06839aeafd59fe0550c554af0e9f/packages/extension/src/shared/token/storage.ts#L30

Comment (h0ngcha0) [reply packages/extension/src/shared/token/storage.ts:55]: Yes, i will add `description` as well. Thanks!

File: packages/extension/src/shared/token/storage.ts
Comment (polarker) [inline packages/extension/src/shared/token/storage.ts:8]: Let's just migrate the old version whenever we can, like in this case. A new version would bring additional complexity in the long term because we will need to manage all old versions.

File: packages/extension/src/shared/token/storage.ts
Comment (polarker) [inline packages/extension/src/shared/token/storage.ts:31]: There is no need to have this field now?

Comment (polarker) [reply packages/extension/src/shared/token/storage.ts:31]: It's part of the type, we cannot omit. Let's keep it then

**Review by polarker — APPROVED** [review]
The code LGTM! Once tests pass, we merge and release.

**Pull request merged**
